schemas/xml/README.md

```diff
@@ -0,0 +1,38 @@
+# XML 
+In the following repository an overview of the main concepts of the AssetAdministration Shell XML serialization is presented. For import and export scenarios the metamodel of an AssetAdministration Shell needs to be serialized. A serialization format is XML. The information is divided in three parts. The first part discusses the rules, in the second part are examples for some specific rules and in the third part the schema and a complete example is shown in the annex.
```

Comment:
For import and export scenarios the metamodel of the Asset Administration Shell needs to be serialized. A serialization format is XML.